evm_loader/lib/src/commands/get_config.rs

```diff
@@ -291,5 +354,35 @@ pub async fn read_chains(
 ) -> NeonResult<Vec<ChainInfo>> {
     let mut simulator = rpc.build_config_simulator(program_id).await?;
 
-    simulator.get_chains(program_id).await
+    simulator.get_chains().await
+}
+
+impl CloneRpcClient {
+    async fn get_account_with_sol(&self) -> ClientResult<Pubkey> {
+        let r = self
+            .get_largest_accounts_with_config(RpcLargestAccountsConfig {
+                commitment: Some(self.commitment()),
+                filter: None,
+            })
+            .await?; // TODO: This is very inefficient and needs to be removed
```

Comment:
Personally I'm not sure what to do with such comments. There are no immediate plans to fix that, right? I suppose the comment will stay forever unless the code is totally rewritten. 🙃 Perhaps it makes sense to create an issue if we really want to address this at some point.

Reply:
There is an issue here: https://neonlabs.atlassian.net/browse/NDEV-2462